### PR TITLE
Changelog for 8.11.0.

### PR DESCRIPTION
**Pull request by Zimmi48**
**Kind:** documentation / meta.

As requested by @ppedrot. Note that if more PRs with changelog entries are backported (like #11329 or #11355), this PR will need to be amended (or another one opened if it has already been merged) so better merge it late in the process.

### CODE REVIEW

**Comment by cpitclaudel** [comment]
LGTM, let me know when the merge should happen

**Comment by Zimmi48** [comment]
Since this is directly related to the release, I think that @ppedrot can take care of merging this PR. Thanks @cpitclaudel!

**Comment by ppedrot** [comment]
@Zimmi48 this will need amending, but it is likely that one more backport is going to make it into 8.11, namely the sub-PR of #11355 that is self-contained. Will you have time for this to happen quickly when needed?

**Comment by Zimmi48** [comment]
Time is not a problem as this will be easy to amend. Though, if you want me to react quickly, you should probably reach out to me directly by e-mail, as I won't be monitoring my GitHub notifications (I've got a paper to write).

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -55,23 +55,23 @@ this version of Coq, it should soon be the case and we already
...
+Coq's documentation is available at https://coq.github.io/doc/v8.11/api (documentation of
+the ML API), https://coq.github.io/doc/v8.11/refman (reference
+manual), and https://coq.github.io/doc/v8.11/stdlib (documentation of
 the standard library).
 
 The OPAM repository for |Coq| packages has been maintained by
-Karl Palmskog, Matthieu Sozeau, Enrico Tassi with contributions
-from many users. A list of packages is available at
+Guillaume Claret, Karl Palmskog, Matthieu Sozeau, Enrico Tassi with
+contributions from many users. A list of packages is available at
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst]: Added Guillaume's name here following a suggestion from @palmskog.

**Comment by ppedrot** [comment]
@SkySkimmer can you rebase this? Hopefully we're going to backport it today.

**Comment by Zimmi48** [comment]
I'm on it.

**Comment by ppedrot** [comment]
@Zimmi48 thanks a lot. I don't think we need full CI to complete, refman and test-suites should suffice.

**Review by ppedrot — APPROVED** [review]

**Pull request merged**